src/commands/push-to-app-collection.yaml

```
@@ -0,0 +1,41 @@
+parameters:
+  app_name:
+    type: "string"
+  app_catalog:
+    type: "string"
+  app_collection_repo:
+    type: "string"
+steps:
+  - run: |
+        architect project version > version
+  - run: |
+        echo -n "<<parameters.app_name>>-$(cat version)" | head -c 53 > cr-name
```

Same thing about hidden files.

```suggestion
        echo -n "<<parameters.app_name>>-$(cat version)" | head -c 53 > .cr-name
```